### PR TITLE
updated hyperlink text

### PR DESCRIPTION
**Pull request by lavernw1**
@inoxx03 pls review. thanks.

### CODE REVIEW

File: docs/topics/minishift-install-start-configure-minishift.adoc [hunk 1]
```diff
@@ -42,7 +42,7 @@ Since a {OpenShiftLocal} is for development purposes, it uses HTTPS for the web
 
 image::minishift_sslwarning.png[SSL warning]
 
-At this time, Safari on macOS does not work with a {OpenShiftLocal}. Details on this issue are tracked link:https://github.com/openshiftio/appdev-planning/issues/16[here]. Other browsers, such as Chrome, on macOS do work with a {OpenShiftLocal}.
+At this time, Safari on macOS does not work with a {OpenShiftLocal}. This a known issue. Details are tracked at link:https://github.com/openshiftio/appdev-planning/issues/16[known issue]. Other browsers, such as Chrome, on macOS do work with a {OpenShiftLocal}.
```
Comment (Ladicek) [inline docs/topics/minishift-install-start-configure-minishift.adoc]: Weird wording. Maybe `This is a link:...[known issue].`?

Comment (inoxx03) [inline docs/topics/minishift-install-start-configure-minishift.adoc]: @lavernw1 I agree. as @Ladicek  pointed out, the link text should fit smoothly into the text, if possible. I'd suggest rephrasing the sentence as:

> At this time, Safari on macOS does not work with a {OpenShiftLocal}. This a link:https://github.com/openshiftio/appdev-planning/issues/16[known issue].

Comment (lavernw1) [inline docs/topics/minishift-install-start-configure-minishift.adoc]: yes i agree. thanks for the feedback @Ladicek @inoxx03

**Comment by inoxx03** [comment]
LGTM, merging...

**Comment by inoxx03** [comment]
merged ad1d1d894d0926d6eb369d221b9088f9ed3ef13f

**Pull request merged**